overlord/snapstate/backend.go

```diff
@@ -25,6 +25,7 @@ import (
 	"golang.org/x/net/context"
 
 	"github.com/snapcore/snapd/asserts"
+	"github.com/snapcore/snapd/client"
```

Comment:
This feels slightly strange - makes me wonder if something like `store/buy` would be better but then it would be a tiny tiny package so probably not quite right?